style.css

```diff
@@ -50,7 +63,7 @@
 }
 
 /* .about-me-tags > * {
-  margin: 8px 0;
+  margin: 8px 0;   contact-us-side-img.svg
 } */
```

Comment:
- [ ] Please remove the comments if it is not necessary for your code. They will make your project look messy.